Viper Practice/src/networking/API.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  API.swift
+//  NetworkLayer
+//
+//  Created by Malek TRABELSI on 08/01/2023.
+//
+
+import Foundation
+
+class API {
```

Comment:
why not make it a protocol?

Reply:
create APIProtocol and add getFruites function in the protocol